src/main/java/propertyFiles/Bundle.properties

```diff
@@ -1322,6 +1322,8 @@ dataset.locked.ingest.message=The tabular data files uploaded are being processe
 dataset.unlocked.ingest.message=The tabular files have been ingested.
 dataset.locked.editInProgress.message=Edit In Progress
 dataset.locked.editInProgress.message.details=Additional edits cannot be made at this time. Contact {0} if this status persists.
+dataset.locked.pidNotReserved.message=Dataset Locked
+dataset.locked.pidNotReserved.message.details=The DOI displayed in the citation for this dataset has not yet been registered with DataCite. Please do not share this DOI until it has been registered.
```

Comment:
I know we don't want to keep going back and forth on this, but... this msg reads like we need additional success and warning issues if the DOI was finally registered or ultimately changed. This came to mind reading the comment from @landreev in Slack in regards to issue #6980.

Reply:
I don't feel strongly about the wording and don't mind if someone changes it.